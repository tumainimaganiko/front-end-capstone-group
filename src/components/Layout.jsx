import { Outlet } from 'react-router-dom';
import NavigationPanel from './NavigationPanel';
import Log from './log/log';

const Layout = () => (
  <>
    <NavigationPanel />
    <Log />
    <Outlet />
  </>
);

export default Layout;
