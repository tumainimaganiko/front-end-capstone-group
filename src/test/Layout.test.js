import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Layout from '../components/Layout';

it('Render a Layout', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Layout />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
