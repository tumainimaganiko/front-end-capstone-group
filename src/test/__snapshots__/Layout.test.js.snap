// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a Layout 1`] = `
Array [
  <button
    className="fixed z-50 text-white md:top-5 md:left-5"
    onClick={[Function]}
    type="button"
  >
    <svg
      aria-hidden="true"
      className="text-[34px]"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 20 20"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M3 7a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 13a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
        fillRule="evenodd"
      />
    </svg>
  </button>,
  <button
    className="hidden"
    type="button"
  >
    Login
  </button>,
]
`;
